provider/openstack/cinder.go

```diff
@@ -107,7 +107,7 @@ var newOpenstackStorage = func(env *Environ) (OpenstackStorage, error) {
 	client := env.clientUnlocked
 	if env.volumeURL == nil {
 		url, err := getVolumeEndpointURL(client, env.cloudUnlocked.Region)
-		if errors.IsNotFound(err) {
+		if errors.IsNotFound(err) || gooseerrors.IsNotFound(err) {
```

Comment:
Same here - can you make a function that returns true for both juju/errors NotFound and goose NotFound?